cmd/soroban-cli/src/lib.rs

```diff
@@ -14,6 +14,7 @@ pub use cli::main;
 pub mod assembled;
 pub mod commands;
 pub mod config;
```

Comment:
Why not move this into `config`? Then you can do `use crate::config::dir_check;`